tf2_kdl/docs/source/conf.py

```diff
@@ -11,7 +11,7 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import roslib
+#import roslib
```

Comment:
Does this code work as is? If so, remove references to ROS 1 instead of commenting them out